cmd/backup-operator/README.md

```diff
@@ -0,0 +1,104 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3]. Currently, backup operator only supports saving backup to AWS S3.
+
+## Requirements
+* Kubernetes 1.7+
+* etcd 3.1+
+
+## Getting Started
+
+You can try out etcd backup operator by running it on `Kubernetes` and then create a `Backup Custom Resource` which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `Backup Custom Resource`, retrieves etcd data, and then saves data as a backup to S3.
```

Comment:
```
`Backup Custom Resource` -> `EtcdBackup` Custom Resource
```

retrieves etcd data -> retrieves etcd snapshots

saves data as a backup to S3 -> saves it to S3